tests/middleware/test_wsgi.py

```
@@ -0,0 +1,129 @@
+    start_response(status, headers)
+    return [output]
+
+
+def raise_exception(environ, start_response):
+    raise RuntimeError("Something went wrong")
+
+
+def return_exc_info(environ, start_response):
+    try:
+        raise RuntimeError("Something went wrong")
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)